gh-125682: Python implementation of `json.loads()` accepts non-ascii digits (#125687)
python/cpython: Misc/NEWS.d/next/Library/2024-10-18-09-51-29.gh-issue-125682.vsj4cU.rst

> Reject unicode digits for Python implementation of :func:`json.loads`.

Reviewer: I would clarify that this is a bug. Something more like "Conform to RFC 8259's number specification for Python implementation of `json.loads`"

> Reject unicode digits for Python implementation of :func:`json.loads`.

Reviewer 2: Unicode includes the digits 0-9 at code points 0x30 thru 0x39, so a different description should be used rather than "Unicode digits". I suggest "non-ASCII" which is short and simple.
```suggestion
Reject non-ASCII digits in the Python implementation of :func:`json.loads`.
```

Reject non-ASCII digits in the Python implementation of :func:`json.loads`

> conforming to the json specification.

Reviewer: Nitpick:
```suggestion
conforming to the JSON specification.
```

Author: Yeah, it became lowercase when I translated the sentence with DeepL.
